frontend/packages/ux-editor/src/hooks/mutations/useAddPageToGroup.test.ts

```
@@ -0,0 +1,226 @@
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { app, org, selectedLayoutSet } from '@studio/testing/testids';
+
+const mockSetSelectedFormLayoutName = jest.fn();
+const mockUpdateLayoutsForPreview = jest.fn();
+
+describe('useAddPageToGroup', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it.only('should add a page to the group and call the mutation', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)